src/content/docs/synthetics/synthetic-monitoring/private-locations/verified-script-execution-private-locations.mdx

```diff
@@ -27,13 +27,13 @@ To enable verified script execution for containerized private minions, do the fo
 1. Go to **[one.newrelic.com](https://one.newrelic.com/) > Synthetics > Private locations > (select a private location)**. Select the private location's ellipses icon, and click **Edit**. Enable verified script execution, then save.
 2. Set the passphrase in your Docker or Kubernetes environment:
 
-   * **Docker:** Add the `MINION_VSE_PASSWORD` environment variable to the Docker `run` command used to start your private minion:
+   * **Docker:** Add the `MINION_VSE_PASSWPHRASE` environment variable to the Docker `run` command used to start your private minion:
```

Comment:
PASSPHRASE

Reply:
@kaileyhaynes, this should be fixed, now!